.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -0,0 +1,12 @@
+name: "Validate Gradle Wrapper"
+on: [push]
```

Comment:
Without `pull_request`, this won't actually run for external contributors

Reply:
Are you sure this is accurate?

Reply:
Pretty sure. Push only applies to branches on the project, and your PR is not a branch on the project. We had to fix this in our autodispose project as well when enabling actions

Reply:
You were right. Should be fixed.
Thanks!